stdlib/string.ml

```diff
@@ -40,6 +40,7 @@ let empty = ""
 let of_bytes = B.to_string
 let to_bytes = B.of_string
 let sub s ofs len =
+  if ofs = 0 && length s == len then s else
```

Comment:
Please use `length s = len` or `Int.equal (length s) len`, see #13573.

Reply:
Too much python these days ;-) Thanks
By the way, length s = len should be enough. if ofs is not zero, it will raise invalid_arg in sub ?